cura/UI/MachineSettingsManager.py

```diff
@@ -0,0 +1,72 @@
+
+from typing import Any, Dict, Optional, TYPE_CHECKING
+
+from PyQt5.QtCore import Qt, QObject, pyqtSlot
+
+from UM.i18n import i18nCatalog
+
+
```

Comment:
Documentation; What is the responsibility of this class?

Reply:
Added in 82471a0e3